README.rst

```diff
@@ -20,8 +20,12 @@ Jupyter Server Proxy
    :target: https://www.npmjs.com/package/@jupyterlab/server-proxy
 
 Jupyter Server Proxy lets you run arbitrary external processes (such
-as RStudio, Shiny Server, syncthing, PostgreSQL, etc) alongside your
-notebook, and provide authenticated web access to them.
+as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc)
+alongside your notebook server and provide authenticated web access to
+them using a path like `/rstudio` next to others like `/lab`.
+Alongisde the python package that provide the main functionality, the
+JupyterLab extension (`@jupyterlab/server-proxy`) provide buttons in
```

Comment:
```suggestion
JupyterLab extension (`@jupyterlab/server-proxy`) provides buttons in
```
This is an RST file so I don't think the single backticks do what you intended (format as code?). You could try double backticks but run a sphinx/rst build to verify.